### DISCUSSION TITLE
How to get markdown always to display the source code？

### DISCUSSION THREAD

**Discussion by gengdz** [LazyVim/LazyVim#269] [Q&A] [answered]
The markdown source code is displayed on the line where the cursor is located, and the rest seems to be parsed automatically

![image](https://user-images.githubusercontent.com/41912568/218991428-08c516d1-2757-4786-87cb-7f0c2baf9951.png)
![image](https://user-images.githubusercontent.com/41912568/218991589-3f9da030-3612-430c-98fa-31a7a6d93875.png)

**Comment by folke** [comment, marked as answer]
Add the below to your `lua/config/autocmds.lua` file:

```lua
vim.api.nvim_create_autocmd({ "FileType" }, {
  pattern = { "markdown" },
  callback = function()
    vim.wo.conceallevel = 0
  end,
})
```

Alternatively, if you don't want to disable conceal for all filetypes, just add the below to `lua/config/options.lua` instead:

```lua
 vim.wo.conceallevel = 0
```

**Reply by gengdz** [reply]
That works, thank you so much
